packages/core/src/org.ts

```diff
@@ -31,16 +31,27 @@ export enum OrgRole {
      * Member information (like public key and email address) of suspended members
      * are considered unverified, and need to be updated and verified via a
      * membership confirmation [[Invite]].
+     * @deprecated Use `OrgMemberStatus.Suspended` instead
      */
     Suspended,
 }
 
+export enum OrgMemberStatus {
+    Provisioned = "provisioned",
```

Comment:
This seemed like a more ergonomic way to "provision" new organisation members, rather than storing them on the `OrgProvisioning` object.